src/controller.ts

```diff
@@ -1,15 +1,16 @@
 import { Application } from "pixi.js";
 import { Minigame, MinigameDelegate } from "./minigames/minigame";
-import { KeyboardMinigame } from "./minigames/keyboardMinigame";
+import {DragMinigame } from "./minigames/dragMinigame";
 
 /** Controls the flow of the game. */
 export class Controller implements MinigameDelegate {
   private currentMinigame?: Minigame;
 
   constructor(private readonly app: Application) { }
 
-  start() {
-    this.currentMinigame = new KeyboardMinigame(this.app, this);
+  async start() {
+    this.currentMinigame = new DragMinigame(this.app, this);
```

Comment:
After you merge/rebase in changes from #12 (if it happens to land before yours), you'll want to add DragMinigame to the list of ALL_MINIGAMES and `await preload` in the `populateMinigameQueue` method, I reckon. If this lands before #12 no action required

Reply:
Actually I just realised this minigame has no win or lose condition so it locks players in lol. Maybe we should leave it out of ALL_MINIGAMES to begin with